cmd_test.go

```diff
@@ -22,29 +26,51 @@ const (
 	cmdExampleArgs = "arg1 arg2"
 )
 
+func waitMessages(t *testing.T, count int, errorCount int) {
+	for {
+		select {
+		case reply := <-replies:
+			msgs = append(msgs, reply)
+		case err := <-cmdError:
+			errs = append(errs, err)
+		case <-time.After(1 * time.Second):
+			t.Fatal("Timed the shit out")
+		}
+		if len(msgs) == count && len(errs) == errorCount {
```

Comment:
Can these equal signs be made to be >= ? And or add a check for them being too large, and fail the test inside this method if either one grows beyond it's expected length.